students/Sally_Shen/lesson01/assignment/inventory_management/electric_appliances_class.py

```diff
@@ -0,0 +1,30 @@
+"""Electric appliances class"""
+from .inventory_class import Inventory
```

Comment:
When running, you will likely be running from the assignment directory, you will want to update this to:
from inventory_management.inventory_class import Inventory